src/serializers/type_serializers/bytes.rs

```diff
@@ -4,24 +4,28 @@ use pyo3::prelude::*;
 use pyo3::types::{PyBytes, PyDict};
 
 use crate::definitions::DefinitionsBuilder;
+use crate::serializers::config::BytesMode;
 
 use super::{
     infer_json_key, infer_serialize, infer_to_python, BuildSerializer, CombinedSerializer, Extra, SerMode,
     TypeSerializer,
 };
 
 #[derive(Debug, Clone)]
-pub struct BytesSerializer;
+pub struct BytesSerializer {
+    bytes_mode: BytesMode,
```

Comment:
I think this probably needs to be `Option<BytesMode>`, and reflect what got passed in the config at serializer build time.